criu/Makefile

```diff
@@ -21,6 +21,9 @@ CONFIG-DEFINES		+= -DUSER_CONFIG_DIR='".criu/"'
 
 #
 # General flags.
+ifeq (gcc, $(findstring gcc, $(CC)))
+    CFLAGS += -fno-reorder-blocks-and-partition
+endif
```

Comment:
Could you try this: https://github.com/checkpoint-restore/criu/pull/1749 ?
Disabling optimization in the whole project because of a minor issue in PIEs would be overkill.
At worst, you could have disabled it for a specific restorer's file.

Reply:
OK, I'll try this.

> At worst, you could have disabled it for a specific restorer's file.

I agree, but I just don't know how to modify the Makefile to implement this. I'm not that familiar with Makefile.

Reply:
> Could you try this: https://github.com/checkpoint-restore/criu/pull/1749 ?

Quite strange because the .cold symbol is still generated. Maybe I should modify the Makefile and disable it for `criu/pie/restorer.c` but I have no idea how to do this.